build-bin/docker_args

```diff
@@ -14,30 +14,36 @@ DOCKER_ARCHS="amd64 arm64"
 
 case "${JRE_VERSION}" in
   6 )
+    DOCKER_BUILD_IMAGE=ghcr.io/openzipkin/java:11.0.22_p7
     DOCKER_PARENT_IMAGE=ghcr.io/openzipkin/java:1.6.0-119
     # single arch image
     DOCKER_ARCHS=amd64
     ;;
   7 )
+    DOCKER_BUILD_IMAGE=ghcr.io/openzipkin/java:11.0.22_p7
     DOCKER_PARENT_IMAGE=ghcr.io/openzipkin/java:1.7.0_285
     # single arch image
     DOCKER_ARCHS=amd64
     ;;
   8 )
+    DOCKER_BUILD_IMAGE=ghcr.io/openzipkin/java:11.0.22_p7
     DOCKER_PARENT_IMAGE=ghcr.io/openzipkin/java:8.392.08-jre
     ;;
   11 )
+    DOCKER_BUILD_IMAGE=ghcr.io/openzipkin/java:11.0.22_p7
     DOCKER_PARENT_IMAGE=ghcr.io/openzipkin/java:11.0.22_p7-jre
     ;;
   17 )
+    DOCKER_BUILD_IMAGE=ghcr.io/openzipkin/java:21.0.2_p13
```

Comment:
had to parameterize this as spring boot requires minimal JRE 17.. the first project to require this.